client/template.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title> Silverspoon! </title>
+</head>
+<link rel="stylesheet" type="text/css" href="about-styles.css">
+<link rel="stylesheet" type="text/css" href="reviews.css">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+<body id="mainPage">
+  <div>
+      <span class="title"> Silverspoon! </span>
+    <span>
+      <img class="logo" src="https://avatars0.githubusercontent.com/u/36716959?s=200&v=4"></img>
+    </span>
+  </div>
+  <div id="summary"></div>
+  <div id="reservations"></div>
+  <div id="menu"></div>
+  <div id="reviews"></div>
+  <script src="https://fb.me/react-0.14.3.min.js"></script>
+  <script src="https://fb.me/react-dom-0.14.3.min.js"></script>
```

Comment:
should these be a newer version of React, or is 14 working ok for you?

Reply:
oh, i shouldve waited before doing this branch but my previous PR added the unpkg 16.2 version of react, and when I created a new branch frmo master, it had the old react cdn